### ISSUE TITLE
[DOCUMENT]: GIBCT SCO - Page headings SHOULD be tagged in VA ONCE PDF

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8750]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

VA 508 office has identified the [VA Once Quick Reference Guide](https://www.benefits.va.gov/GIBILL/docs/vaonce/VAONCEguide.pdf) PDF as not having any page headers. I've attached a screenshot showing a sample page with a `<p>` tag that could be changed into an `<h2>` for page headings.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/3917

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Remediation Resources
* Run the [Common Look PDF validator](https://commonlook.com/accessibility-software/pdf-validator/) or the [Acrobat Accessibility Checker](https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html) to check for headers being tagged properly

## Acceptance Criteria

* Page headers (h2) are announced properly

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs
![Screen Shot 2020-05-05 at 9 58 30 AM](https://user-images.githubusercontent.com/934879/81081438-8503ce80-8eb7-11ea-8b61-000384bfb72b.png)

**Comment by desireeturner** [comment]
@TamGIBill I believe this is content that you can update. Please let me know if it is not. Thank you!

**Comment by noahgelman** [comment]
@af-manish Waiting on the update from @TamGIBill as the previous comment states

**Comment by af-manish** [comment]
@noahgelman <cc @ERRonat0609 @mrblanco1 > Gentle nudge here.

**Comment by ERRonat0609** [comment]
@af-manish @mrblanco1 - Why are we asking Noah for another update when, as of 17 days ago, he responded that we are waiting on an update from @TamGIBill. Tammy has been assigned to work on this.

**Comment by af-manish** [comment]
@TamGIBill I wanted to follow-up to see if you can revisit this.

**Comment by mrblanco1** [comment]
Tammy will follow up at the end of March 2022.

**Comment by mrblanco1** [comment]
This ticket does not belong to AFS Comparison Tool. @TamGIBill might be able to provide a solution. Thanks!

**Comment by TamGIBill** [comment]
There is a VA 508 initiative that will resolve this.

Tammy S. Hurley
Education Service
Veterans Benefits Administration
202-461-9838

From: Abe Blanco ***@***.***>
Sent: Wednesday, September 14, 2022 4:06 PM
To: department-of-veterans-affairs/va.gov-team ***@***.***>
Cc: Hurley, Tammy S. (she/her/hers)(she/they) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [department-of-veterans-affairs/va.gov-team] [DOCUMENT]: GIBCT SCO - Page headings SHOULD be tagged in VA ONCE PDF (#8750)


This ticket does not belong to AFS Comparison Tool. @TamGIBill<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTamGIBill&data=05%7C01%7C%7Cf3606d8715f9427c823b08da968c95f0%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637987827803443762%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=cH3OmmD32lkpCg0go8KaUweTnBw6NpqOEHxAIrgGat0%3D&reserved=0> might be able to provide a solution. Thanks!

-

**Comment by TamGIBill** [comment]
There is a VA 508 initiative that will resolve this.


Tammy S. Hurley
Education Service
Veterans Benefits Administration
202-461-9838

From: Abe Blanco ***@***.***>
Sent: Wednesday, September 14, 2022 4:06 PM
To: department-of-veterans-affairs/va.gov-team ***@***.***>
Cc: Hurley, Tammy S. (she/her/hers)(she/they) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [department-of-veterans-affairs/va.gov-team] [DOCUMENT]: GIBCT SCO - Page headings SHOULD be tagged in VA ONCE PDF (#8750)


This ticket does not belong to AFS Comparison Tool. @TamGIBill<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTamGIBill&data=05%7C01%7C%7Cf3606d8715f9427c823b08da968c95f0%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637987827803443762%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=cH3OmmD32lkpCg0go8KaUweTnBw6NpqOEHxAIrgGat0%3D&reserved=0> might be able to provide a solution. Thanks!

-

**Comment by joshkimux** [comment]
@TamGIBill do we know of any updates since? @artsymartha68 is this something we can follow up with the 508 office on? Until then, marking as blocked

**Comment by TamGIBill** [comment]
@joshkimux @artsymartha68  VA ONCE has sunset with the implementation of Enrollment Manager.  These training documents will be removed shortly.  Thanks.

**Comment by joshkimux** [comment]
Closing this out!
